### DISCUSSION TITLE
python version

### DISCUSSION THREAD

**Discussion by jjhesk** [FuelLabs/fuels-ts#1857] [Ideas]
Is this possible to have a python version of the wallet-sdk?

**Comment by arboleya** [comment]
There are no plans for this.

**Comment by jjhesk** [comment]
are there any documentations for the implementation of fuel wallet in text?

**Comment by arboleya** [comment]
You can take a look at the specs:
 - https://github.com/FuelLabs/fuel-specs

The Wallet Chrome Extension implementation:
 - https://github.com/FuelLabs/fuels-wallet

And also, the Wallet CLI implementation:
 - https://github.com/FuelLabs/forc-wallet
